openstack/nova/files/compute.nova.conf

```diff
@@ -44,24 +45,18 @@ neutron_url = http://{{ virl.controller_ip }}:9696
 neutron_auth_strategy = keystone
 neutron_admin_tenant_name = service
 neutron_admin_username = neutron
-neutron_admin_password = {{salt['pillar.get']('virl:password', salt['grains.get']('password', 'password')) }}
+neutron_admin_password = {{ virl.ospassword }}
 neutron_admin_auth_url = http://{{ virl.controller_ip }}:35357/{{ virl.keystone_auth_version }}
 security_group_api = nova
 
 #Hey raaabbbit!
 rabbit_host = {{ virl.controller_ip }}
-rabbit_password = {{salt['pillar.get']('virl:password', salt['grains.get']('password', 'password')) }}
+rabbit_password = {{ virl.ospassword }}
 
-my_ip = {{  salt['pillar.get']('virl:internalnet_ip', salt['grains.get']('static_ip', '127.0.0.1' )) }}
-
-#vnc section
-vncserver_listen = {{  salt['pillar.get']('virl:static_ip', salt['grains.get']('static_ip', '127.0.0.1' )) }}
-vncserver_proxyclient_address = {{  salt['pillar.get']('virl:static_ip', salt['grains.get']('static_ip', '127.0.0.1' )) }}
-novncproxy_port = {{ salt['pillar.get']('virl:vnc_port', salt['grains.get']('vnc_port', 19407)) }}
-novncproxy_base_url = http://127.0.1.1:{{ salt['pillar.get']('virl:vnc_port', salt['grains.get']('vnc_port', '19407')) }}/vnc_auto.html
+my_ip = {{ virl.int_ip }}
 
 ram_allocation_ratio = 5.0
-scheduler_default_filters= {{  salt['pillar.get']('virl:nova_filter', salt['grains.get']('nova_filter', 'AllHostsFilter' )) }}
+scheduler_default_filters={{ salt['pillar.get']('virl:nova_filter', salt['grains.get']('nova_filter', 'RamFilter,AllHostsFilter,ComputeFilter' )) }}
```

Comment:
There had been issues with the filters in the past, but I do think they should work fine now.